scanner/scanner.go

```diff
@@ -588,7 +587,6 @@ func (s *Scanner) scanLiteralHeader(ctx *Context) (pos int, err error) {
 				ctx.literalOpt = opt
 				return
 			}
-			break
```

Comment:
This "break" is unnecessary because it's at the end of the switch statement. I wonder if it was actually meant to be placed below to break out of the for loop 🤔